cli.sh

```diff
@@ -51,6 +52,56 @@ print_help_header() {
     echo "----------------------------------------------"
 }
 
+#####################################################
+# Check if a path is an absolute path
+# Returns:
+#   0 if absolute, 1 if not
+#####################################################
+is_abs_path() {
+    [ "$1" != "${1#/}" ] || return 1
+}
+
+#####################################################
+# Gets the absolute path of a path
+#   reference:
+#   https://unix.stackexchange.com/a/24342/63957
+#####################################################
+make_abs_path() {
+    if ! is_abs_path "$1"; then
+        if [ -d "$1" ]; then f="$1/."; fi
+        # shellcheck disable=SC2164
+        absolute=$(
+            cd "$(dirname -- "$f")"
+            printf %s. "$(pwd)"
+        )
+        absolute="${absolute%?}"
+        echo "$absolute/${1##*/}"
+    else
+        echo "$1"
+    fi
+}
+
+#####################################################
+# Set the current install dir.
+# This script location determines it.
+#
+# Sets the ${INSTALL_DIR} variable.
+#
+# Returns:
+#   0 if suncessful, 1 if not
+#####################################################
+set_install_dir() {
```

Comment:
Let's bin `set_install_dir`, `make_abs_path` and `is_abs_path` for now, and just go with `~/.githooks/tools`

Reply:
SLOA?